src/dlgPackageExporter.cpp

```diff
@@ -582,3 +800,21 @@ void dlgPackageExporter::listTimers()
         recurseTimers(pChild, pItem);
     }
 }
+
+void dlgPackageExporter::displayResultMessage(const QString& html, bool const isSuccessMessage)
+{
+    if (!isSuccessMessage) {
+        ui->infoLabel->setText(QStringLiteral("<p><font color='red'><b><big>%1</big><b></font></p>").arg(html));
+        return;
+    }
+
+    ui->infoLabel->setText(tr("<p><b><big>%1</big><b></font></p>"
+                              "<p>Why not <a href=\"https://forums.mudlet.org/posting.php?mode=post&f=6\">upload</a> your package to Mudlet?</p>",
```

Comment:
This may produce `You need to login in order to post within this forum.` if not logged in already. Maybe link the sub-forum's overview instead, where logged in users may click `New topic` button then, and others may at least read and ponder to register?

Reply:
I was merely reusing the existing URL - changing that is something I would happily defer to another PR...! :stuck_out_tongue_winking_eye:

Reply:
I see. The diff made me think this was a new string, but it was merely moved around some. :bulb: